### PR TITLE
Update Race Status

### PR DESCRIPTION
**Pull request by iole-geil**
Update `race.status` with validations

### CODE REVIEW

**Review by joppetburger — APPROVED** [review]
+1 yung mga unless lang ule

**Comment by iole-geil** [comment]
> +1 yung mga unless lang ule

Resolved in this [branch/commit](https://github.com/scrambledeggs/iole-trails/pull/7/commits/0dccf19cd079450fd6e4ebba5a2ddf0eefcca7ee)

File: app/views/races/show.html.erb [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
-<%= link_to "Remove", trail_race_path([@trail, @race]), data: {
-                    turbo_method: :delete,
-                    turbo_confirm: "Are you sure?"
-                  } %>
\ No newline at end of file
+<% if @race.status_NEW? %>
+<%= form_with model: [@trail, @race] do |form| %>
+  <%= form.hidden_field :status, value: :STARTED %>
+  <%= form.submit "Start Race"%>
+<% end %>
```
Comment (alecguintu) [inline app/views/races/show.html.erb:17]: Better indent this so you know it's inside the if condition

File: app/views/races/show.html.erb [hunk 2]
```diff
@@ -1,12 +1,38 @@
...
 <p>Start: <%= @race.start.to_fs(:db) %></p>
 <p>Duration (hours): <%= @race.duration %></p>
 
-<%= link_to "Edit Details", edit_trail_race_path([@trail, @race]) %> | 
-<%= link_to "Remove", trail_race_path([@trail, @race]), data: {
-                    turbo_method: :delete,
-                    turbo_confirm: "Are you sure?"
-                  } %>
\ No newline at end of file
+<% if @race.status_NEW? %>
```
Comment (alecguintu) [inline app/views/races/show.html.erb:13]: You can also use `<%- %>` with `-` so it does not create a space upon printing the html. Check on inspect elements. :)

**Review by alecguintu — CHANGES_REQUESTED** [review]

File: app/views/races/show.html.erb (hunk 1 above)
Comment (iole-geil) [reply app/views/races/show.html.erb:17]: Updated in [this commit](https://github.com/scrambledeggs/iole-trails/pull/7/commits/59df0b846c001ec185c33f73038c1832e3bdf0a9)

File: app/views/races/show.html.erb (hunk 2 above)
Comment (iole-geil) [reply app/views/races/show.html.erb:13]: Based on discussion, I will accommodate this feedback in a different branch and PR later.

**Review by alecguintu — APPROVED** [review]

**Pull request merged**